client/signup/main.jsx

```diff
@@ -598,8 +598,10 @@ class Signup extends React.Component {
 		return flows.getFlow( flowName, this.props.isLoggedIn ).steps.indexOf( stepName );
 	}
 
-	getFlowLength() {
-		return flows.getFlow( this.props.flowName, this.props.isLoggedIn ).steps.length;
+	getInteractiveStepsCount() {
+		const flowStepsSlugs = flows.getFlow( this.props.flowName, this.props.isLoggedIn ).steps;
+		const flowSteps = flowStepsSlugs.filter( ( step ) => ! steps[ step ].props.nonInteractive );
```

Comment:
Won't this throw when `props (optional)` doesn't exist?
```suggestion
		const flowSteps = flowStepsSlugs.filter( ( step ) => ! steps[ step ].props?.nonInteractive );
```

Reply:
Thanks! Awesome catch!